portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,8 +36,7 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
```

Comment:
nit - update this comment to remove the TODO (since it's done!)

Reply:
Ok.